src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -312,9 +312,11 @@ class RsErrorAnnotator : Annotator, HighlightRangeExtension {
         val traitRef = impl.traitRef ?: return
         val trait = traitRef.resolveToTrait ?: return
         val traitName = trait.name ?: return
-        if (impl.unsafe != null && trait.unsafe == null) {
+        if (impl.unsafe != null && impl.excl != null) {
+            holder.createErrorAnnotation(traitRef, "Negative implementations are not unsafe [E0198]")
```

Comment:
The already existing errors are displayed on the trait name. So i mimicked that behavior.
rustc seems to display the error on the whole line instead for E0198, E0199 and E0200.

But i guess thats not a huge deal.